train/util/util.py

```diff
@@ -4,7 +4,6 @@
 import torch
 import torch.nn as nn
 
-
```

Comment:
I generally (but sometimes fail to) follow pep8 style guidelines. According to the [guidelines](https://www.python.org/dev/peps/pep-0008/#blank-lines), top-level functions should be surrounded by two blank lines.

I have an old script that will auto-format to fix these. I'll add that at a later time, but, for now, let's revert this back to two blank lines.

Reply:
Yes. Sure! I missed it. I'll squash and send a new commit.